milvus/MilvusClient.ts

```diff
@@ -92,9 +93,29 @@ export class MilvusClient extends GRPCClient {
       throw new Error(ERROR_REASONS.CREATE_COLLECTION_CHECK_PARAMS);
     }
 
-    // if fields are in the data, use old _createCollection
-    if ('fields' in data) {
-      return await this._createCollection(data);
+    // if fields or schema are in the data, use old _createCollection
+    if ('fields' in data || 'schema' in data) {
+      const createCollectionRes = await this._createCollection(data);
+
+      // if index params available
+      if ('index_params' in data) {
+        // creata index
+        for (let indexParam of data.index_params) {
+          const i = await this.createIndex(
+            Object.assign(indexParam, { collection_name: data.collection_name })
+          );
+        }
```

Comment:
should we use Promise.all instead? For better performance.
Also we can delete `const i =` if useless.